qa/tasks/mgr/dashboard/test_pool.py

```diff
@@ -150,7 +151,7 @@ def test_pool_create_fail(self):
         self.assertJsonBody({
             'component': 'pool',
             'code': "2",
-            'detail': "specified rule dnf doesn't exist"
+            'detail': "[errno -2] specified rule dnf doesn't exist"
```

Comment:
this should address the failure of http://pulpito.ceph.com/kchai-2018-05-12_08:15:46-rados-wip-kefu-testing-2018-05-12-1405-distro-basic-smithi/2527167/